Configuration-Interaction/CI_DL.py

```diff
@@ -117,20 +112,14 @@
 for CI_ITER in range(max_guess - 1):
 
     # Subspace Matrix, Gij = < bi | H | bj >
-    #for i in range(num_vecs - num_eig, num_vecs):
     for i in range(0, num_vecs):
         # Build sigma for each b
         ciwfn.sigma(cvecs, svecs, i, i)
```

Comment:
@dgasmith @tyzhang1993 For some reason, the sigma vectors for roots other than the ground state are totally wrong on the first iteration. It does seem to give the right sigma vector for new subspace vectors added at each iteration. Not sure why the behavior is different at the first iteration.